experimenter/experimenter/experiments/tests/factories.py

```diff
@@ -114,6 +114,7 @@ class NimbusVersionedSchemaFactory(factory.django.DjangoModelFactory):
     )
     schema = factory.LazyAttribute(lambda o: FAKER_JSON_SCHEMA)
     sets_prefs = factory.LazyAttribute(lambda o: [])
+    is_early_startup = factory.LazyAttribute(lambda o: False)
```

Comment:
Oh you can just say 

```py
is_early_startup = False
```

for static values like that, you should only need the lambda for computed values?